rotator/rotator.py

```diff
@@ -16,8 +16,10 @@
 EXCLUDE_CHARACTERS = r'''"%!'()*,/:;?@[\]`{|}~<>^&#$'''
 
 CLIENT_REGISTRY_TEMPLATE = URITemplate('https://clients.oauth.cimpress.io/v1/clients/{client_id}/secrets')
+ID_KEY = os.environ.get('CLIENT_ID_KEY') if os.environ.get('CLIENT_ID_KEY') is not None else "id"
```

Comment:
Here and below should probably use the 2-arg version of `get`. Also, house style is single-quoted strings.

Reply:
Great call, I'm rusty with my python - forgot about get with the default value. Pushed out the changes